apple/OmnivoreKit/Sources/App/AppExtensions/Share/ShareExtensionScene.swift

```diff
@@ -41,11 +42,16 @@ final class ShareExtensionViewModel: ObservableObject {
   }
 
   func savePage(extensionContext: NSExtensionContext?) {
+    backgroundTask = UIApplication.shared.beginBackgroundTask(withName: "BACKGROUND")
```

Comment:
Should we add a UUID to this? Could be possible to have two background jobs 🤷🏽‍♂️

Reply:
yeah, definitely. Maybe we generate the clientRequestId right at the start and pass this all the way through.